Bottom: e9865551aae1c331a5afba06832f7e41103176bf
Top:    00678bb2636f6259f0e22981aebd0fc3b8b1c947
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-04 14:39:24 +0000

Refresh of add-macros-to-access-next-and

---

diff --git a/src/polygon1.c b/src/polygon1.c
index c6e7739..9753048 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1084,10 +1084,11 @@ print_labels (PLINE * a)
 
   do
     {
-      DEBUGP ("%#mD->%#mD labeled %s\n", c->point[0], c->point[1],
-	      NEXT_VERTEX (c)->point[0], NEXT_VERTEX (c)->point[1], theState (c));
+      DEBUGP ("%#mD->%#mD labeled %s\n",
+              EDGE_BACKWARD_VERTEX (c)->point[0], EDGE_BACKWARD_VERTEX (c)->point[1],
+               EDGE_FORWARD_VERTEX (c)->point[0],  EDGE_FORWARD_VERTEX (c)->point[1], theState (c));
     }
-  while ((c = NEXT_VERTEX (c)) != &a->head);
+  while ((c = NEXT_EDGE (c)) != &a->head);
 }
 #endif
 #endif
